Makefile

```diff
@@ -2414,6 +2414,45 @@ datasets/sqwiki.autolabeled_revisions.20k_2016.json: \
 		--verbose > \
 	datasets/sqwiki.autolabeled_revisions.20k_2016.json
 
+datasets/sqwiki.human_labeled_revisions.5k_2016.json:
+	./utility fetch_labels \
+		https://labels.wmflabs.org/campaigns/sqwiki/57/ > \
+	datasets/sqwiki.human_labeled_revisions.5k_2016.json
+
+datasets/sqwiki.human_labeled_revisions.5k_2016.no_review.json: \
+		datasets/sqwiki.human_labeled_revisions.5k_2016.json
+	cat datasets/sqwiki.human_labeled_revisions.5k_2016.json | \
+	grep '"needs_review": false' > \
+	datasets/sqwiki.human_labeled_revisions.5k_2016.no_review.json
+
+datasets/sqwiki.autolabeled_revisions.20k_2016.no_review.json: \
+		datasets/sqwiki.autolabeled_revisions.20k_2016.json
+	cat datasets/sqwiki.autolabeled_revisions.20k_2016.json | \
+	grep '"needs_review": false' > \
+	datasets/sqwiki.autolabeled_revisions.20k_2016.no_review.json
+
+datasets/sqwiki.labeled_revisions.20k_2016.json: \
+		datasets/sqwiki.human_labeled_revisions.5k_2016.no_review.json \
+		datasets/sqwiki.autolabeled_revisions.20k_2016.no_review.json \
+		datasets/sqwiki.human_labeled_revisions.5k_2016.json
+	( \
+	  ./utility merge_labels \
+	    datasets/sqwiki.human_labeled_revisions.5k_2016.no_review.json \
+	    datasets/sqwiki.autolabeled_revisions.20k_2016.no_review.json; \
+	  cat datasets/sqwiki.human_labeled_revisions.5k_2016.json | \
+	  grep '"needs_review": true' | shuf -rn 4558 \
```

Comment:
These notes apply to the whole makefile, no fault of your patch.

shuf -r seems strange to me--is there a reason to allow repeated rows?  

I'm curious about the magic number 4558, it seems to be copied from cswiki but isn't that going to be dependent on the actual size of the merge_labels output?  This seems like a slight anti-pattern anyway, because we shouldn't need to know about the output before writing the rule.  How about instead,
```
( \
	  ./utility merge_labels \
	    datasets/sqwiki.human_labeled_revisions.5k_2016.no_review.json \
	    datasets/sqwiki.autolabeled_revisions.20k_2016.no_review.json; \
	  cat datasets/sqwiki.human_labeled_revisions.5k_2016.json | \
	    grep '"needs_review": true' | shuf \
	) | head -n 20000 > datasets/sqwiki.labeled_revisions.20k_2016.json
```
(Although, I'm not sure we need shuf at all.)